stdlib/http/src/main/java/org/ballerinalang/net/http/HttpUtil.java

```diff
@@ -441,6 +443,7 @@ public static HttpCarbonMessage createErrorMessage(String payload, int statusCod
         if (payload != null) {
             payload = lowerCaseTheFirstLetter(payload);
             response.addHttpContent(new DefaultLastHttpContent(Unpooled.wrappedBuffer(payload.getBytes())));
+            response.setHeader(X_CONTENT_TYPE_OPTIONS, NO_SNIFF);
```

Comment:
lets add a comment explaining why we did this.

Reply:
Added a comment 👍🏼